plugins/mcreator-core/triggers/command_executed.json

```diff
@@ -21,8 +21,12 @@
       "type": "number"
     },
     {
-      "name": "command",
```

Comment:
You should also still provide command to be backwards compatible

Reply:
Oops, I used the refactoring feature and I did not notice it.